### PR TITLE
(test) O3-2112:Add Add notes e2e test

### PR DESCRIPTION
**Pull request by RandilaP**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
Goal of this PR is to adding a add notes e2e test to patient chart repository

## Related Issue

https://issues.openmrs.org/browse/O3-2112

### CODE REVIEW

File: e2e/specs/add-note.spec.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+let patient: Patient;
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+});
+
+test('should be able to add a note to a patient', async ({ page, api }) => {
+  const notesPage = new NotesPage(page);
+
+  await test.step('When I visit the notes page', async () => {
```
Comment (Piumal1999) [inline e2e/specs/add-note.spec.ts:14]: @RandilaP, We might need to update the gherkin syntax keywords of theses steps. Please refer to [this tutorial](https://www.guru99.com/gherkin-test-cucumber.html#4). 

Here's a small overview of how to use the Gherkin keywords:
image

Comment (Piumal1999) [reply e2e/specs/add-note.spec.ts:14]: We use the "And" keyword when 2 or more consecutive steps with the same starting word come up.

Example:
```
When I input the email address
When I input the correct password
```

The above can be replaced with:
```
When I input the email address
And I input the correct password
```

File: e2e/specs/add-note.spec.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    await notesPage.startVisit();
+    await expect(notesPage.page.getByText('Active Visit')).toBeVisible();
+  });
+
+  await test.step('And I click Record visit notes', async () => {
+    await notesPage.page.getByText('Record visit notes').click();
+  });
+
+  await test.step('And I fill a note', async () => {
+    await notesPage.page.locator('#visitDateTimePicker').fill('03/07/2023');
```
Comment (Piumal1999) [inline e2e/specs/add-note.spec.ts:28]: Can we use 'getByLabel' or 'getByRole' functions for querying this (instead of using the id)?

Comment (Piumal1999) [reply e2e/specs/add-note.spec.ts:28]: Same for line 29,31,33

**Review by Piumal1999 — CHANGES_REQUESTED** [review]

**Comment by RandilaP** [comment]
@anjula-sack @Piumal1999 @jayasanka-sack Hey guys @ibacher said that this feature intentionally removed and we don't need this test anymore. So can I close this PR then?

**Comment by anjula-sack** [comment]
> @anjula-sack @Piumal1999 @jayasanka-sack Hey guys @ibacher said that this feature intentionally removed and we don't need this test anymore. So can I close this PR then?

sure. can you link the slack thread here and close this. @RandilaP

**Comment by RandilaP** [comment]
https://openmrs.slack.com/archives/C02UNMKFH8V/p1692206732465129

**Comment by RandilaP** [comment]
https://openmrs.slack.com/archives/C02UNMKFH8V/p1692206732465129

**Pull request closed without merging**